day2b.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+with open("input2a.txt", "r") as fp:
+    lines = fp.read().strip().splitlines()
+
+sum_power = 0
+for line in lines:
+    game_str, remainder = line.split(": ")
+    game_id = int(game_str.split()[-1])
+    game = remainder.replace("; ", ", ").split(", ")
+    
+    max_colors = {'red': 0, 'green': 0, 'blue': 0}
```

Comment:
If we want to be generic to be able to handle any arbitrary colors, we can do that with a `defaultdict`. First, `from collections import defaultdict`, then:
```suggestion
    max_colors = defaultdict(int)
```

Reply:
Pretty neat! thanks